aiconfig-docs/docs/getting-started.md

```diff
@@ -159,223 +200,192 @@ asyncio.run(main())
 import * as path from "path";
 import { AIConfigRuntime, InferenceOptions } from "aiconfig";
 
-async function travelWithGPT() {
-  // Load the AIConfig. You can also use AIConfigRuntime.loadJSON({})
-  const aiConfig = AIConfigRuntime.load(
-    path.join(__dirname, "travel.aiconfig.json")
-  );
-
-  // Run a single prompt
-  const result = await aiConfig.run("get_activities");
-  console.log(result);
-}
-
-travelWithGPT();
+const aiConfig = AIConfigRuntime.load(
```

Comment:
For node, this will need to be within a function that's called at the bottom of the file:

```
import * as path from "path";
import { AIConfigRuntime, InferenceOptions } from "aiconfig";

async function app() {
  const aiConfig = AIConfigRuntime.load(
    path.join(__dirname, "travel.aiconfig.json")
  );
}

app();
```

See previous getting-started code, or https://github.com/lastmile-ai/aiconfig/blob/b9ef01fcec3aadc64d8d29fe6596209c4dadee3c/cookbooks/llama/typescript/ask-llama.ts#L4 for example

Reply:
Updated with the typescript code from the previous getting-started code thanks!